internal/testdata/objectives.go

```diff
@@ -39,6 +51,28 @@ func genericDFO() directfund.Objective {
 		Nonce:             ts.ChannelNonce.Int64(),
 		Outcome:           ts.Outcome,
 	}
-	testObj, _ := directfund.NewObjective(request, false)
+	testObj, err := directfund.NewObjective(request, false)
+	if err != nil {
+		panic(fmt.Errorf("error constructing genericDFO: %w", err))
+	}
 	return testObj
 }
+
+func genericVFO() virtualfund.Objective {
+	ts := testVirtualState.Clone()
+	request := virtualfund.ObjectiveRequest{
+		ts.Participants[0],
+		ts.Participants[2],
+		ts.Participants[1],
```

Comment:
This is a little jarring to read... does it hint at a small refactor that we should do?

Reply:
addressed in 2b09634. Jarring indeed.